test/src/components/__snapshots__/book-tile.test.tsx.snap

```diff
@@ -31,6 +31,7 @@ exports[`book-tile renders 1`] = `
       >
         <button
           aria-controls="sample-ddm"
+          aria-expanded="false"
```

Comment:
I can't quite tell what change added this, maybe a test changed the open state? Just confirming if it is expected.

Reply:
Previously, the default value of `aria-expanded` was null. Converting to TS made me use boolean, so it's explicit rather than absent.